File: beginner/javascript/2534.js
function bubbleSort(arr) {
    const n = arr.length;
    let temp;

    for (let i = 0; i < n - 1; i++) {
        for (let j = 0; j < n - 1 - i; j++) {
            if (arr[j] < arr[j + 1]) {
                temp = arr[j];
                arr[j] = arr[j + 1];
                arr[j + 1] = temp;
            } 
        }
    }
}

const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question((input) => {
    const [n, q] = input.split(' ').map(Number);
    rl.question((numbersInput) => {
        const numbers = numbersInput.split(' ').map(Number);

        bubbleSort(numbers);

        for (let i = 0; i < q; i++) {
            rl.question((positionInput) => {
                const position = parseInt(positionInput);

                if (position >= 1 && position <= n) 
                    console.log(numbers[position - 1]);

                if (i === q - 1) 
                    rl.close();
            });
        }
    });
});

File: beginner/python/2534.py
def bubble_sort(arr):
    n = len(arr)
    for i in range(n - 1):
        for j in range(n - 1 - i):
            if arr[j] < arr[j + 1]:
                arr[j], arr[j + 1] = arr[j + 1], arr[j]

while True:
    try:
        n, q = map(int, input().split())
        numbers = list(map(int, input().split()))

        bubble_sort(numbers)

        for i in range(q):
            position = int(input())
            if 1 <= position <= n:
                print(numbers[position - 1])
            else:
                print("Posição inválida")

    except EOFError:
        break
